### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by AnAverageBeing**
Changes
1. added -threads option
2. renamed many things for better understanding
3. made a better showcase video
4. updated README

### CODE REVIEW

**Comment by Z3NTL3** [comment]
# Request feedback
Hello!

Thanks for making out an pull request. However after a review we marked this pull request as unnecessary and inefficient.

For example, you have changed our ``switch`` statement which was used to check input with an very long ``if`` condition. It is a bad practice to use a ``if`` statement on long conditions instead of ``switch``. A ``switch`` statement uses more less memory in the situation.

We also see that you rename variable names which is also not needed. Everything is already self-explaining. 

You have also accomplished doing inefficient functionalities. Like adding arguments to a function, as these parameter you defined were already drawn through the ``globals``.

So basically we won't be pulling this fork however thanks for your efforts. I appreciated your interest in ProxyBeast <3

# Additional information
### Future enhancement deadlines
- Multi functionality will be added on (``3 feb 2023``) -> ``https://github.com/Z3NTL3/ProxyBeast/issues/5``
